dist/css/datepicker.css

```diff
@@ -187,10 +187,9 @@
   font-size: 14px;
   color: #4a4a4a;
   width: 250px;
-  position: absolute;
```

Comment:
Why are you removing position absolute?

Reply:
The position: absolute was not being applied because in Frappe, the datepicker.scss had been overwritten to apply position: fixed to the .datepicker class. I reverted it back to position: absolute to ensure that the datepicker behaves correctly by default, even if someone hasn't overwritten the default styles.